daemons/gptp/common/avbts_port.hpp

```diff
@@ -426,7 +426,7 @@ class IEEE1588Port {
 	 * hardware timestamper and create OS locks conditions
 	 * @return FALSE if error during building the interface. TRUE if success
```

Comment:
Please create an entry here for the delay fields. From the interface point of view, its not clear what is the meaning of each member of the array 'delay' (one gets that by reading the code)